mp/src/game/server/momentum/tickset.cpp

```diff
@@ -96,7 +123,51 @@ bool TickSet::SetTickrate(Tickrate trNew)
     return false;
 }
 
-static void OnTickRateChange(IConVar *var, const char* pOldValue, float fOldValue)
+
+static void OnTickRateSet(const CCommand &command)
+{
+    constexpr unsigned rateCount = sizeof(TickSet::s_DefinedRates) /
+        sizeof(*TickSet::s_DefinedRates);
+
+    // Search defined rates for one with a string matching the command argument.
+    for (unsigned rateI = 0; rateI < rateCount; rateI++)
+    {
+        if (!strcmp(command.ArgS(), TickSet::s_DefinedRates[rateI].sType))
+        {
+            TickSet::SetTickrate(TickSet::s_DefinedRates[rateI].fTickRate);
+            return;
+        }
+    }
+    Warning("Unknown tickrate. Use sv_interval_per_tick to set custom tickrates.");
+}
+
+static int OnTickRateAutocomplete(const char* partial,
+    char commands[COMMAND_COMPLETION_MAXITEMS][COMMAND_COMPLETION_ITEM_LENGTH])
+{
+    constexpr unsigned rateCount = sizeof(TickSet::s_DefinedRates) /
+        sizeof(*TickSet::s_DefinedRates);
+
+    // Move start of the string to expected start of parameter. (null char counted for space)
+    partial += sizeof("sv_tickrate");
```

Comment:
The command name `sv_tickrate`'s length will be constant here, you can just hard code that and put a comment saying what it is:
```suggestion
    partial += 12; // strlen("sv_tickrate") + 1
```